### PR TITLE
Update dotprompt and eval doc to reflect menuSuggestionFlow theme

### PR DESCRIPTION
**Pull request by MaesterChestnut**

### CODE REVIEW

File: docs/dotprompt.md [hunk 1]
```diff
@@ -164,22 +164,22 @@ output:
     description: string
 ---
 
-Generate a tabletop RPG character that would be found in {{location}}.
+Generate a restaurant menu that would be found in {{location}}.
```
Comment (MichaelDoyle) [inline docs/dotprompt.md:167]: This sample is showcasing the ability to coerce the LLM to provide complex JSON output. As such, the schema above (L159-164) specifies an output object with 3 fields `name`, `hitPoints`, `description.`.

If we want to go with the menu theme, we should update the schema with a few different properties, and perhaps tweak the prompt.

We could do something like:

```
---
model: vertexai/gemini-1.0-pro
input:
  schema:
    theme: string
output:
  format: json
  schema:
    name: string
    price: integer
    ingredients(array): string
---

![image](https://github.com/firebase/genkit/assets/2858322/579f29e9-427d-4419-bbce-b2895dba74bf)


Generate a menu item that could be found at a {{theme}} restaurant.
```

File: docs/firebase.md [hunk 2]
```diff
@@ -123,9 +123,9 @@ deploying the default sample flow to Firebase.
     next section), in the `httpsOptions` parameter, set a CORS policy:
```
Comment (MichaelDoyle) [inline docs/firebase.md:123]: Please remove this file, in favor of #112.

File: docs/evaluation.md [hunk 3]
```diff
@@ -55,7 +59,7 @@ Then navigate to `localhost:4000/evaluate`.
 Alternatively, you can provide an output file to inspect the output in a json file.
 
 ```posix-terminal
-genkit eval:flow bobQA --input testQuestions.json --output eval-result.json
```
Comment (MichaelDoyle) [inline docs/evaluation.md:58]: There is another reference to `bobQA` much further down below, under "Running on existing datasets"

**Review by MichaelDoyle — CHANGES_REQUESTED** [review]

File: docs/dotprompt.md (hunk 1 above)
Comment (MichaelDoyle) [reply docs/dotprompt.md:167]: Prompt in action. 

image

**Review by kevinthecheung — APPROVED** [review]

**Review by MichaelDoyle — APPROVED** [review]

**Pull request merged**